flixel/addons/display/FlxBackdrop.hx

```diff
@@ -29,6 +30,9 @@ class FlxBackdrop extends FlxSprite
 	private var _spaceX:Int = 0;
 	private var _spaceY:Int = 0;
 	
+	private var _colorTransform:ColorTransform;
```

Comment:
Apparently yes! I tested it with the project I uploaded in the previous message and another project I'm working on. No errors encountered.

Do I apply the changes?